### DISCUSSION TITLE
Can we remove the permissions to delete the logs

### DISCUSSION THREAD

**Discussion by bathivinod** [opcodesio/log-viewer#408] [Ideas]
Can we remove the permissions to delete the logs

![image](https://github.com/user-attachments/assets/6ccb2be5-af98-4aa3-81f6-e68279a696f9)

**Comment by MrYamous** [comment]
Documention explain how to [authorize (or not) log file deletion](https://log-viewer.opcodes.io/docs/3.x/configuration/access-to-log-files#authorizing-log-file-deletion)

**Reply by arukompas** [reply]
That is correct! Thanks @MrYamous 

Locking discussion, as there's no other answer :)
